float32 probability
PointXYRGB[] linePoints
